.pipelines/templates/android_shared_lib_build.yml

```diff
@@ -0,0 +1,70 @@
+# packaging pipeline for onnxruntime-extensions Android AAR package
+parameters:
+- name: AndroidABI
+  displayName: android abi,[arm64-v8a, x86_64, X86, armeabi-v7a]
+  values:
+  - 'x86_64'
+  - 'x86'
+  - 'arm64-v8a'
+  - 'armeabi-v7a'
+  type: string
+- name: Android_arch
+  displayName: same android abi, but in valid var
+  type: string
+
+jobs:
+  - job: Android_C_API_Packaging_${{ parameters.Android_arch }}
+    pool:
+      vmImage: "macOS-12"
+    timeoutInMinutes: 120
+    variables:
+      buildConfig: Release
+    steps:
+      - task: UsePythonVersion@0
+        inputs:
+          versionSpec: "3.9"
+          addToPath: true
+          architecture: "x64"
+        displayName: "Use Python 3.9"
+
+      - task: JavaToolInstaller@0
+        displayName: Use jdk 11
+        inputs:
+          versionSpec: "11"
+          jdkArchitectureOption: "x64"
+          jdkSourceOption: "PreInstalled"
+
+      - script: brew install coreutils ninja
+        displayName: Install coreutils and ninja
+
+      - script: |
+          python ./tools/gen_selectedops.py ./tools/android/package_ops.config
```

Comment:
we want to build all ops and set OCOS_ENABLE_SELECTED_OPLIST=OFF, is that right?

Reply:
Currently the request is to build the libraries for the nuget package with `-DOCOS_ENABLE_CV2=OFF -DOCOS_ENABLE_BLINGFIRE=OFF` rather than selected ops list.

Reply:
Got it.